pkg/controllers/v1alpha1/databackup/implement.go

```diff
@@ -350,10 +400,17 @@ func (r *DataBackupReconcilerImplement) generateDataBackupValueFile(ctx reconcil
 	}
 
 	if len(imageTag) == 0 {
-		imageTag = docker.GetImageTagFromEnv(common.ALLUXIO_RUNTIME_IMAGE_ENV)
+		if runtimeType == "alluxio" {
+			imageEnv = common.ALLUXIO_RUNTIME_IMAGE_ENV
+			defaultImage = common.DEFAULT_ALLUXIO_RUNTIME_IMAGE
+		} else if runtimeType == "goosefs" {
+			imageEnv = common.GOOSEFS_RUNTIME_IMAGE_ENV
+			defaultImage = common.DEFAULT_GOOSEFS_RUNTIME_IMAGE
+		}
+		imageTag = docker.GetImageTagFromEnv(imageEnv)
 		if len(imageTag) == 0 {
-			defaultImageInfo := strings.Split(common.DEFAULT_ALLUXIO_RUNTIME_IMAGE, ":")
-			if len(defaultImageInfo) < 2 {
+			defaultImageInfo := strings.Split(defaultImage, ":")
+			if len(defaultImageInfo) < 1 {
 				panic("invalid default databackup image!")
```

Comment:
Does it have to be a panic?

Reply:
tks, i think it is necessary be panic.

Reply:
Here if it's panic and what we see  ?  pod restart ?